api/nakadi-event-bus-api.yaml

```diff
@@ -484,3 +752,114 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Event'
+
+  EventType:
+    description: An event type defines the schema and its runtime properties.
+    properties:
+      name:
+        type: string
+        description: Name of this EventType. The name for the EventType must follow the pattern
+        'organization:team:application:event-type'
+      topic:
+        type: string
+        description: |
+          References the name of the low-level topic that is holding the stream of Events for this EventType.
+          Usually is the same as the name property.
+      validation-strategies:
+        description: |
+          Determines the validation that has to be executed upon reception of Events of this type. Events are
+          rejected if any of the rules fail (see details of Problem response on the Event publishing methods).
+
+          Rule evaluation order is the same as in this array.
+        type: array
+        items:
+          $ref: '#/definitions/EventValidationStrategy'
```

Comment:
I currently don't understand how all these strategies are defined. Will there just be a set of predefined strategies like some enum ?